@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/interact/fn_interactBattSwitch.sqf

```diff
@@ -0,0 +1,46 @@
+/* ----------------------------------------------------------------------------
+Function: fza_systems_fnc_interactBattSwitch
+
+Description:
+	Sets button state for the APU sim.
+
+Parameters:
+	_heli   - The helicopter to get information from [Unit].
+
+Returns:
+	Whether to register a click (boolean).
+
+Examples:
+	...
+
+Author:
+	BradMick
+---------------------------------------------------------------------------- */
+params ["_heli"];
+
+private _battSwitchOn = _heli getVariable "fza_systems_battSwitchOn";
+
+if (_battSwitchOn) then {
+	_battSwitchOn = false;
+} else {
+	_battSwitchOn = true;
+};
+_heli setVariable ["fza_systems_battSwitchOn", _battSwitchOn, true];
+/*
+switch (_battSwitchOn) do {
```

Comment:
Please can you remove the commented out code?

Reply:
Fixed.